Level-Editor.md

```diff
@@ -9,3 +9,73 @@ There are various videos on the internet documenting the use of the built-in lev
 - [Advanced Tutorial (Part 1/2)](https://www.youtube.com/watch?v=WBdwwcLD-vw) by Oliver Buo
 - [Advanced Tutorial (Part 2/2)](https://www.youtube.com/watch?v=UoaGDuBax6E) by Oliver Buo
 - [Expert Tutorial](https://www.youtube.com/watch?v=lL3oZbPfw08) by Oliver Buo
+
+## Share Levels
+
+When you finished creating a level or even more, there is an easy way to share it to the SuperTux community.
```

Comment:
@manuel-st @maxteufel Isn't it "share it with"?

Reply:
Would say that too. Share it with the SuperTux community.